data/data/azure/bootstrap/main.tf

```diff
@@ -152,6 +152,8 @@ resource "azurerm_virtual_machine" "bootstrap" {
 }
 
 resource "azurerm_network_security_rule" "bootstrap_ssh_in" {
+  count = var.preexisting_network ? 0 : 1
```

Comment:
I think we can drop this as NSG is always created.

Reply:
Dropped.